looper.py

```diff
@@ -0,0 +1,242 @@
+import os
```

Comment:
Hmmm did this file get duplicated? We had moved it into `audnauseum/state_machine/looper.py`

Reply:
Yeah, I'll remove that.